packages/salesforcedx-vscode-apex/src/commands/metadataOrchestrator.ts

```diff
@@ -50,8 +58,8 @@ export class MetadataOrchestrator {
    */
   public extractMethodMetadata = (): MethodMetadata | undefined => {
     const editor = vscode.window.activeTextEditor;
-    if (!editor) {
-      notificationService.showErrorMessage(nls.localize('no_active_editor'));
+    if (!editor || !editor.document.fileName.endsWith('cls')) {
```

Comment:
The command won't show up anyways if it's not a .cls file so I think we don't need this

Reply:
Agreed but just for double check in case extractMethodMetadata is used in other commands in future.